README.md

```diff
@@ -1 +1,219 @@
+- [ ] **[Display_to do_List]**
+- [ ] **[Mark as Done To do list]**
+- [ ] **[Refactor using ES6]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> You can use the project by using the following steps.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to access the link of the project.
+
+https://github.com/Yonasle/todo-list-app
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+cd Desktop/Microverse https://github.com/Yonasle/todo-list-app
+
+git clone https://github.com/Yonasle/todo-list-app
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project to your local repo:
+
+
+cd Desktop/Microverse
+code .
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+
+cd Desktop
+live-server
+
+
+### Deployment
+
+You can deploy this project using:
+
+Not available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Yonas**
+
+- GitHub: [@githubhandle](https://github.com/Yonasle)
+- Twitter: [@twitterhandle](https://twitter.com/Yonasls)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yonas-lemma-b8848823a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> The following are the features of the project
+
+- [ ] **[CSS layout]**
+- [ ] **[Add link to each feature]**
+- [ ] **[Using recent update of EcmaScript Feautres]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project , Kindly Please give me ⭐. Thank you
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+We will like to thanks the student succes and Microverse coordinators for they give us such important task/project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Also, make sure you fix the link to your LICENSE to make your README more professional 👍 

![Screenshot 2023-08-17 171648](https://github.com/Yonasle/todo-list-app/assets/57408419/86666a48-0c68-464c-9b29-9021714ac6cb)